lib/Backend/Commands.hs

```diff
@@ -91,7 +91,19 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
+    checkForEntriesInEntryPath entryPath = do
+      let eParentDirs = forM ((NE.tail . Path.entryPathParentDirs) entryPath) Path.pathAsEntryPath
+      case eParentDirs of
+        Right parentDirs -> filterM pathIsEntry parentDirs >>= \case
+          (clashed : _) -> throw $ CRParentPathContainsEntry clashed
```

Comment:
I think we should return and report all clashing paths, not just the first one.

Reply:
I think in this case if we've found clashed entry, then it would be the only one (all parents would be definitely directories).